docs/UserGuide.md

```diff
@@ -278,42 +263,42 @@ Examples:
 * `findPolicy cmp/PRU` returns policies that belong to Prudential Assurance Company
 * `findPolicy ti/Shield cov/LIFE` returns polices with Shield in its title and covers the LIFE coverage type
 
-
 ### Assigning a Policy to a Client 
 
 If you have successfully sealed a deal with a client, you can keep track of this by assigning the policy to your client! 
 
 If you have yet to add either your Client or Policy to the Application, you can refer to the [Adding a Person](#adding-a-person-add) or [Adding a Policy](#adding-a-policy) guides.
 
+### Tracking your Income  : `viewIncome`
 
-### Tracking your Income 
-
-
-
-## Event Features 
+This allows you to sort your clients in the address book based on a specified metric.
 
+Format: `viewIncome <year>`
```

Comment:
i think this might be a good place to add a UI example so we have more diagrams in our UG?